app/src/main/res/xml/pref_data_sync.xml

```diff
@@ -8,9 +8,24 @@
         android:title="@string/data_sync">
         <PreferenceScreen
             android:icon="@drawable/ic_qrscan"
-            android:key="auto_configure"
+            android:key="auto_config_top"
             android:summary="@string/prefs_auto_config_summary"
-            android:title="@string/auto_configure_title" />
+            android:title="@string/auto_configure_title">
+            <PreferenceScreen
+                android:key="auto_configure"
+                android:summary="@string/auto_config_cam_summary"
+                android:title="@string/auto_config_cam_title" />
+            <Preference
```

Comment:
Can't we just have these one underneath each other in the same way as other preferences?

Reply:
Definitely!  I can do that.

The only reason I did this was because I wanted to give more prominence to the existing scheme. I mean if user can use camera, they should never use the second option.
 That's the only reason I did this.

Reply:
I have made both of those changes.

This is what it looks like now:
![Screenshot_20240802-131859](https://github.com/user-attachments/assets/0b830720-35df-4f12-9544-2458c53e3f3c)

Reply:
I'll probably keep the QR code on my storage somewhere and almost always use that instead of Camera. That way I don't have to hunt for the code every time I need to reconfigure my app for G7